code/modules/sanity/breakdowns.dm

```diff
@@ -348,79 +348,253 @@
+		"You feel it. You know it. There is no turning back!"
+	)
+		end_messages = list(
+		"You feel at ease again, suddenly."
+	)*/
+	var/time
+	var/cooldown = 15 SECONDS
+	var/time_view = 5 SECONDS
+	var/active_view = FALSE
+	var/mob/living/carbon/human/target
+	start_messages = list("You start to see through everything. Your mind expands.")
+	end_messages = list("The world has returned to normal ... right?")
+
+/datum/breakdown/negative/glassification/can_occur()
+	var/list/candidates = (GLOB.player_list & GLOB.living_mob_list & GLOB.human_mob_list) - holder.owner
+	if(candidates.len)
+		return TRUE
+	return FALSE
+
+/datum/breakdown/negative/glassification/update()
+	if(world.time < time)
+		return TRUE
+	if(active_view)
+		holder.owner.remoteviewer = FALSE
+		holder.owner.remoteview_target = null
+		holder.owner.reset_view(0)
+		target.remoteviewer = FALSE
+		target.remoteview_target = null
+		target.reset_view(0)
+		target = null
+		active_view = FALSE
+		target = null
+		time = world.time + cooldown
+		return TRUE
+	. = ..()
+	if(!.)
+		return FALSE
+	var/list/targets = (GLOB.player_list & GLOB.living_mob_list & GLOB.human_mob_list) - holder.owner
+	if(targets.len)
+		target = pick(targets)
+		holder.owner.remoteviewer = TRUE
+		holder.owner.remoteview_target = target
+		holder.owner.reset_view(target)
+		to_chat(holder.owner, SPAN_WARNING("It seems as if you are looking through someone else's eyes."))
+		to_chat(target, SPAN_WARNING("It seems as if you are looking through someone else's eyes."))
+		target.remoteviewer = TRUE
+		target.remoteview_target = holder.owner
+		target.reset_view(holder.owner)
+		target.sanity.changeLevel(-rand(5,10))
+		active_view = TRUE
+		time = world.time + time_view
+
+/datum/breakdown/common/herald
+	name = "Herald"
+	restore_sanity_pre = 5
+	restore_sanity_post = 45
+	duration = 5 MINUTES
+	start_messages = list("You've seen the abyss too long, and now forbidden knowledge haunts you.")
+	end_messages = list("You feel like you've forgotten something important. But this comforts you.")
+	var/list/messages = list("The Feast is over, and the lamps expire.",
+						"When the stars were right, They could plunge from world to world through the sky; but when the stars were wrong, They could not live.",
+						"In his house at R'lyeh dead Cthulhu waits dreaming.")
+	var/message_time = 0
+
+
+/datum/breakdown/common/herald/can_occur()
+	if(holder.owner.sanity.level <= 20)
+		return TRUE
+	return FALSE
+
+/datum/breakdown/common/herald/update()
+	. = ..()
+	if(!.)
+		return FALSE
+	if(holder.level > 30)
+		finished = TRUE
+		conclude()
+		return FALSE
+	if(world.time >= message_time)
+		message_time = world.time + 10 SECONDS
+		holder.owner.say(pick(messages))
+
+/datum/breakdown/common/desire_for_chrome
+	name = "Desire for Chrome"
+	insight_reward = 30
+	restore_sanity_post = 60
+	start_messages = list("La carne es debil, te repugna la debilidad de tu propio cuerpo.")
```

Comment:
I don't think this is intended

Reply:
no, it is not. It is a text that I put for testing.